helper-cli/src/main/kotlin/common/DatabaseHelper.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.common
+
+import org.jetbrains.exposed.sql.Database
+
+import org.ossreviewtoolkit.model.config.OrtConfiguration
+import org.ossreviewtoolkit.model.config.PostgresStorageConfiguration
+import org.ossreviewtoolkit.model.utils.DatabaseUtils
+import org.ossreviewtoolkit.scanner.LocalScanner
+import org.ossreviewtoolkit.utils.log
+
+internal object DatabaseHelper {
```

Comment:
Is this class actually needed? It does not much more than calling `DatabaseUtils.createHikariDataSource()`. So should this logic be inlined into the command? Or is this function generic enough to be moved to `DatabaseUtils`?

Reply:
I'll move it to a private method in the `DeleteCommand`.